internal/scylla/errors.go

```diff
@@ -7,23 +7,38 @@ import (
 	"strconv"
 )
 
+func IsClusterConnectionDeletedErr(err error) bool {
+	if e := new(APIError); errors.As(err, &e) && e.Code == "083002" {
+		return true
+	}
+	return false
+}
+
 func IsDeletedErr(err error) bool {
 	if e := new(APIError); errors.As(err, &e) && e.Code == "040001" {
 		return true
 	}
 	return false
 }
 
+func IsNotFound(err error) bool {
+	if e := new(APIError); errors.As(err, &e) && e.StatusCode == http.StatusNotFound {
+		return true
+	}
+	return false
+}
+
 // APIError represents an error that occurred while calling the API.
 type APIError struct {
 	URL        string
 	Code       string
 	Message    string
 	Method     string
 	StatusCode int
+	Body       string
```

Comment:
I had a problem with debugging API calls, so I have added it, it was instrumental with my problem.
But I think it could be excessive in usual cases, so I remove it.